app/services/reports/facility_progress_service.rb

```diff
@@ -82,15 +83,88 @@ def dimension_combinations_for(indicator)
 
     attr_reader :diabetes_enabled
 
-    memoize def daily_registrations_grouped_by_day
-      diagnosis = diabetes_enabled ? :all : :hypertension
-      RegisteredPatientsQuery.new.count_daily(facility, diagnosis: diagnosis, last: 30)
```

Comment:
I think we might be able to delete `RegisteredPatientsQuery` now. Check? 💯

Reply:
I think we should keep this around for a while as it is being used in calculations such in finding `monthly_registered_patients_by_gender ` and `monthly_registered_patients_by_user`

Reply:
Let's add a story to our epic to switch those methods over to start using these views. Agreed, it's too much to change all at once in this PR.

Reply:
Bump on removing `count_daily` from `RegisteredPatientsQuery`